orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -47,9 +44,6 @@ public class OrchestrationClient {
     JACKSON.addMixIn(
         ModuleResultsOutputUnmaskingInner.class,
         JacksonMixins.ModuleResultsOutputUnmaskingInnerMixIn.class);
```

Comment:
Why are the mix ins `LLMModuleResult` and `ModuleResultsOutputUnmaskingInner` still there?
It means we are forcing interfaces to deserialize as `synchronous` classes, it can never be `streaming` classes

Reply:
Because https://github.tools.sap/AI/llm-orchestration/issues/778 and https://github.tools.sap/AI/llm-orchestration/issues/779 are not done yet, and deduction doesn't work for streaming